core/src/test/java/org/elasticsearch/search/highlight/HighlightBuilderTests.java

```diff
@@ -362,20 +423,9 @@ private static void setRandomCommonOptions(AbstractHighlighterBuilder highlightB
             highlightBuilder.fragmenter(randomAsciiOfLengthBetween(1, 10));
         }
         if (randomBoolean()) {
-            QueryBuilder highlightQuery;
-            switch (randomInt(2)) {
-            case 0:
-                highlightQuery = new MatchAllQueryBuilder();
-                break;
-            case 1:
-                highlightQuery = new IdsQueryBuilder();
-                break;
-            default:
-            case 2:
-                highlightQuery = new TermQueryBuilder(randomAsciiOfLengthBetween(1, 10), randomAsciiOfLengthBetween(1, 10));
-                break;
-            }
```

Comment:
I can add them back although I don't see the added value of randomized queries here. Adding some random parameters to MatchAll (like boost, name) makes sure we test serialization etc... with some randomness, so that should prevent any hardcoded queries, or what do you mean?